src/static/js/pad_utils.js

```diff
@@ -526,12 +526,27 @@ function setupGlobalExceptionHandler() {
     {
       var errorId = randomString(20);
       var userAgent = padutils.escapeHtml(navigator.userAgent);
-      if ($("#editorloadingbox").attr("display") != "none"){
-        //show javascript errors to the user
-        $("#editorloadingbox").css("padding", "10px");
-        $("#editorloadingbox").css("padding-top", "45px");
-        $("#editorloadingbox").html("<div style='text-align:left;color:red;font-size:16px;'><b>An error occurred</b><br>The error was reported with the following id: '" + errorId + "'<br><br><span style='color:black;font-weight:bold;font-size:16px'>Please press and hold Ctrl and press F5 to reload this page, if the problem persists please send this error message to your webmaster: </span><div style='color:black;font-size:14px'>'"
-          + "ErrorId: " + errorId + "<br>URL: " + padutils.escapeHtml(window.location.href) + "<br>UserAgent: " + userAgent + "<br>" + msg + " in " + url + " at line " + linenumber + "'</div></div>");
+
+      var msgAlreadyVisible = false;
+      $('.gritter-item .error-msg').each(function() {
+        if ($(this).text() == msg) msgAlreadyVisible = true;
```

Comment:
will replace "==" with "==="

Reply:
In these cases, you could also do an early exit once you found a condition that makes the variable true.

In pure JS it would be a `break`. In jQuery there was a way to exit early a `.each()`. I _think_ it was returning `false` from the lambda, but I'd have to look ad the documentation again.

Reply:
Yes I have been lazy :) thanks !